packages/card/src/Card.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { css, cx } from '@leafygreen-ui/emotion';
+import { uiColors } from '@leafygreen-ui/palette';
+
+const containerStyle = css`
+  background-color: white;
+  border: 1px solid ${uiColors.gray.light2};
+  box-shadow: 0px 4px 10px -4px ${uiColors.gray.light1};
+  cursor: pointer;
+
+  &:hover {
+    border: 1px solid ${uiColors.gray.light2};
+    box-shadow: 0px 4px 10px -4px ${uiColors.gray.base};
+  }
+`;
+
+interface CardProps {
+  children: React.ReactNode;
+  as?: React.ElementType<any>;
+  className?: string;
+}
+
+function Card({ children, as = 'div', className, ...rest }: CardProps) {
```

Comment:
The default for `as` of `div` is inconsistent with your README (additionally we might want to default to something more semantic like `<article>`.

Reply:
```<Card
  as='section'
  className='card-styles'
>
  This is my card component
</Card>```

Reply:
just showing how an as work, can default to whatever you feel makes the most sense.